docs/src/pages/components/pagination/examples/ControlledPaginationExample.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Pagination } from '@aws-amplify/ui-react';
+
+export const ControlledPaginationExample = () => {
+  const [currentPageIndex, setCurrentPageIndex] = React.useState(1);
+  const totalPages = 5;
+
+  const handleNextPage = () => {
+    console.log('handleNextPage');
```

Comment:
Why do we have the `console.log`s in this file?

Reply:
To help show the developer when these events are triggered